templates/account/signup.html

```diff
@@ -22,8 +22,9 @@
       </div>
       {% include "socialaccount/snippets/login_extra.html" %}
     {% else %}
-      <p>{% blocktrans %}If you have not created an account yet, then please
-      <a href="{{ signup_url }}">sign up</a> first.{% endblocktrans %}</p>
+      <p>
+        {% trans "accounts.sign_up.login_prompt" default='If you have already created an account, then please <a0>log in</a0>.' tag_a0_href=login_url %}
```

Comment:
By the way, I noticed that `signup_url` was not defined but `login_url` was. Since prompting people to go the page they are seemed unnecessary, I assumed it was a mistake and changed the whole phrase.

Reply:
I think you got this right. These `account`-related templates are a nightmare. We copied them from a dead Django app we still depend on, and then there have been 18 months' worth of changes, mostly untested.